src/modules/UI/scenes/TransactionList/style.js

```diff
@@ -65,15 +65,33 @@ export const styles = {
   },
 
   // end of searchbar stuff
-
+  touchablBalanceBox: {
+    height: 200
+  },
   currentBalanceBox: {
     flex: 1,
     justifyContent: 'center'
   },
+  totalBalanceWrap: {
+    flex: 3,
+    alignItems: 'center',
+    backgroundColor: THEME.COLORS.TRANSPARENT
+  },
+  hiddenBalanceBoxDollarsWrap: {
+    flex: 3,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: THEME.COLORS.TRANSPARENT
+  },
+  currentBalanceBoxHiddenText: {
```

Comment:
what does this do?

Reply:
Since I changed some of the elements in the 'Show Balance' portion of the BalanceBox that means there had to be styling changes.